.teamcity/Gradle_Check/projects/RootProject.kt

```diff
@@ -33,6 +33,8 @@ class RootProject(model: CIBuildModel, functionalTestBucketProvider: FunctionalT
 
     params {
         param("env.GRADLE_ENTERPRISE_ACCESS_KEY", "%ge.gradle.org.access.key%")
+        param("NPM_CONFIG_REGISTRY", "https://repo.gradle.org/gradle/api/npm/npm-temporary/")
+        param("YARN_REGISTRY", "https://repo.gradle.org/gradle/api/npm/npm-temporary/")
```

Comment:
Using `repo.gradle.org` is a temporary measure until we have the new artifactory instance on dev12.

Reply:
Isn't repo.gradle.org as unstable as the npm registry?

Reply:
It's the best we can do for now :crossed_fingers: